eogrow/core/area/batch.py

```diff
@@ -30,15 +30,16 @@ class Schema(BaseAreaManager.Schema):
         )
         resolution: float = Field(
```

Comment:
why not rename this parameter to `tiling_grid_id`, since the user needs to provide the ID anyway?

Reply:
oh no, i borked that up. It is not the ID of the resolution but the actual resolution, it's just constrained :see_no_evil: good catch, will fix

Reply:
Makes sense, but would it still be better if one would provide the grid ID instead of the resolution? seems more intuitive and direct, or does it complicate other things down the line?

Reply:
check L28 ;)

Reply:
🙈